### PR TITLE
Skip unsupported tests from 22.2 runs

### PR DESCRIPTION
**Pull request by akulshre-qa**

### CODE REVIEW

**Comment by mkundu1** [comment]
I have replaced the usage of `FLUENT_IMAGE_TAG` in pytest with markers in #809. Can you please adapt the current PR with those new markers?

**Review by dnwillia-work — APPROVED** [review]

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,4 @@
 [pytest]
-addopts = -m "dev and not fluent_231"
```
Comment (akulshre-qa) [inline pytest.ini:2]: revert the changes from Mainak as this may create confusion when we want to run the whole set of tests.

**Review by prmukherj — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Comment by akulshre-qa** [comment]
> @akulshre-qa I have replaced the usage of `FLUENT_IMAGE_TAG` in pytest with markers in #809. Can you please adapt the current PR with those new markers?

Thanks, Mainak. All those changes are accommodated in this PR.
